extensions/machine-learning/src/views/models/prediction/columnsTable.ts

```diff
@@ -15,7 +15,6 @@ import { ModelParameter, ModelParameters } from '../../../modelManagement/interf
  * View to render azure models in a table
  */
 export class ColumnsTable extends ModelViewBase implements IDataComponent<PredictColumn[]> {
-
 	private _table: azdata.DeclarativeTableComponent | undefined;
 	private _parameters: PredictColumn[] = [];
```

Comment:
I don see any change in models dialog which is correct but when I was testing it, I saw the dialog looks cut off. so probably a regression in core. Please make sure you test regulars  dialogs are working as expected

Reply:
I tested this build on both Windows and Mac and saw no issue with callout used in notebooks and used as data table warning information.